includes/models/model.llms.course.php

```diff
@@ -548,6 +548,15 @@ public function has_prerequisite( $type = 'any' ) {
 
 	}
 
+	/**
+	 * Whether the course lessons must be completed sequentially
+	 *
```

Comment:
Needs `@since` tag.

Class needs `@since` and `@version` updates.

Reply:
The use of @version seems a bit ambiguous, can you define how you want it used?

Reply:
NVM